oauth/static/oauth/prelogin.css

```diff
@@ -0,0 +1,3 @@
+table > thead > tr > th#when {
```

Comment:
I feel like this selector could be simpler, as no one will accidentally use a table header tag outside of a table